concrete/blocks/faq/controller.php

```diff
@@ -69,22 +70,15 @@ public function view()
 
     public function duplicate($newBID)
     {
-        $db = $this->app->make('database')->connection();
```

Comment:
PS: shouldn't we have a `parent::duplicate($newBID);` line in this method?

Reply:
I'm not sure if we need that or not, it seems to work as is